### PR TITLE
docs: link the English document to the official website

### PR DESCRIPTION
**Pull request by Skyenought**
#### What type of PR is this?
docs

#### What this PR does / why we need it (en: English/zh: Chinese):

en: link the English document to the official website
zh: 将英文文档链接指向官网

### CODE REVIEW

**Comment by li-jin-gou** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,71 +1,91 @@
...
 
-## 安装 cwgo 工具
 ```
-# Go 1.15 及之前版本
+# Go version prior to 1.15 
 GO111MODULE=on GOPROXY=https://goproxy.cn/,direct go get github.com/cloudwego/cwgo@latest
 
-# Go 1.16 及以后版本
+
+# Go version after using above than v1.16
```
Comment (GuangmingLuo) [inline README.md:44]: 这句话不通

File: README.md [hunk 2]
```diff
@@ -1,71 +1,91 @@
...
 
-## 开源许可
+### Template Extension
 
-cwgo 基于[Apache License 2.0](https://github.com/cloudwego/cwgo/blob/main/LICENSE) 许可证，其依赖的三方组件的开源许可见 [Licenses](https://github.com/cloudwego/cwgo/blob/main/licenses)。
+Instructions on how to customize templates can be found in the [document](https://www.cloudwego.io/docs/cwgo/tutorials/templete-extension/).
+
+## Open Source License
+
+cwgo has got Apache License2.0 licensing
```
Comment (GuangmingLuo) [inline README.md:74]: cwgo is based on Apache License 2.0

**Comment by li-jin-gou** [comment]
应该是之前的英文有问题，我将 文件名修改了之后这里出现了改变，其实只是加了 link 和默认英文了。



---Original---
From: "Guangming ***@***.***&gt;
Date: Mon, Mar 13, 2023 10:25 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;"State ***@***.***&gt;;
Subject: Re: [cloudwego/cwgo] docs: link the English document to the officialwebsite (PR #48)




 
@GuangmingLuo commented on this pull request.
 
 
In README.md:
 &gt;  GO111MODULE=on GOPROXY=https://goproxy.cn/,direct go get ***@***.***   -# Go 1.16 及以后版本 + +# Go version after using above than v1.16  
这句话不通

**Comment by GuangmingLuo** [comment]
> 应该是之前的英文有问题，我将 文件名修改了之后这里出现了改变，其实只是加了 link 和默认英文了。
> […](#)
> ---Original--- From: "Guangming ***@***.***&gt; Date: Mon, Mar 13, 2023 10:25 AM To: ***@***.***&gt;; Cc: ***@***.***&gt;;"State ***@***.***&gt;; Subject: Re: [cloudwego/cwgo] docs: link the English document to the officialwebsite (PR #48) @GuangmingLuo commented on this pull request. In README.md: &gt; GO111MODULE=on GOPROXY=https://goproxy.cn/,direct go get ***@***.*** -# Go 1.16 及以后版本 + +# Go version after using above than v1.16 这句话不通 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you modified the open/close state.Message ID: ***@***.***&gt;

了解，可以单独提交个 PR 修复下

File: README.md (hunk 2 above)
Comment (Skyenought) [reply README.md:74]: GET

**Pull request merged**